decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/statistics.html.erb

```diff
@@ -1,33 +1,16 @@
 <% if current_participatory_space.private_space? %>
   <%= render partial: "decidim/shared/private_participatory_space", locals: { text: t("participatory_processes.show.private_space", scope: "decidim") } %>
 <% end %>
-
-<div class="row">
-
+<div class="wrapper">
+  <div class="row">
```

Comment:
By design, `row` class encapsulates blocks of 12 columns total. Doing this you are adding all metric block, even if they overflow that 12 columns.